_posts/2021-10-26-no80-0b10G.md

```diff
@@ -0,0 +1,17 @@
+---
+layout: post
+title: Berlin Hack & Tell \#80 - 0b10G
+date: 2021-10-26
+time: '19:30'
+location: '[c-base](https://www.c-base.org)'
+meetupUrl: https://www.meetup.com/de-DE/Berlin-Hack-and-Tell/events/281529520/
+---
+
+* [taxtool](https://github.com/TimDaub/taxtool) by [Tim Daubenschütz](https://github.com/TimDaub)
+* [bikit](https://github.com/lavielle/bikit) by [LaVielle](https://github.com/lavielle) - **Hack of the Month**
+* Package manager from scratch
+* [git prompt shell](https://github.com/jzilg/shellconfig) by [Johannes Zilg](https://github.com/jzilg/shellconfig)
+* IRoam research notebook
+* Educkio by [Marius Kriegerowski](https://github.com/herrmuellerluedenscheid)
+* Solar house monitor
+* [Nix Haskell](https://github.com/qaismyname/nix-haskell) by [Qaiser Abbasi](https://github.com/qaismyname)
```

Comment:
LGTM - ty🙏 it was nice giving the talk btw!

Reply:
thanks for giving the talk!